src/gettingstarted/template/cli-requirements.md

```diff
@@ -0,0 +1,51 @@
+# Start with a template
+
+## Next Steps: Requirements for the CLI
+
+With the [management console](/administration/web.md) you can start new projects from templates just as you did in the previous steps, but deploying your own applications requires you to also use the [Platform.sh CLI](/development/cli.md).
+
+Before you install it, there are a few requirements that must be met first.
+
+### Git
+
+[Git](/development/tools.md#git) is the open source version control system that is utilized by Platform.sh. Any change you make to your Platform.sh project will need to be committed via Git. You can see all the Git commit messages of an environment in the `Environment Activity` feed of the [management console](/administration/web.md) for each project you create.
```

Comment:
"That is utilized by" is far too wordy and passive-voice.  "Used by" or "that Platform.sh uses" would read better.